app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,78 @@
+class TweetsController < ApplicationController
+
+  get '/tweets/new' do
+    if logged_in?
+      erb :'/tweets/create_tweet'
+    else
+      redirect '/login'
+    end
+  end
+
+  post '/create' do
+    if params[:content] != ""
+      @tweet = Tweet.create(content: params[:content])
+      @tweet.user_id = current_user.id
```

Comment:
The tweet creation and user association can both be accomplished in the same step:

``` ruby
@tweet = Tweet.create(content: params[:content], user: current_user)
```